tests/karate/src/test/java/fe/create_lao.feature

```diff
@@ -0,0 +1,25 @@
+@env=android,web
+Feature: Create LAO
+
+  Background: Driver setup
+    * def page_object = 'classpath:fe/utils/<env>.feature'
+    * replace page_object.env = karate.env
+    * call read(page_object)
+
+  Scenario: Create a LAO send the right messages to the backend
+    When click(tab_launch_selector)
+    And input(tab_launch_lao_name_selector, 'Lao Name')
+    And click(tab_launch_create_lao_selector)
+
+    # Retrieving sent messages
+    * json create_lao = backend.takeTimeout(1000)
```

Comment:
For the timeouts i 'll suggest configuring those in the Background 
`configure timeout 1000`